pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -310,7 +313,30 @@ func TestSyncMurStatusWithUserAccountStatusWhenCompleted(t *testing.T) {
 
 		// then
 		require.NoError(t, err)
-		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, toBeProvisioned(), toBeProvisionedNotificationCreated())
+		expectedProvisionedTime := metav1.Now()
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, expectedProvisionedTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
+	})
+
+	t.Run("ProvisionedTime should not be updated when synced more than once", func(t *testing.T) {
+		// given
+		hostClient := test.NewFakeClient(t, mur)
+		sync, memberClient := prepareSynchronizer(t, userAccount, mur, hostClient)
+
+		// when
+		err1 := sync.synchronizeStatus() // 1st sync should not change anything, including ProvisionedTime in MUR
+		initialProvisionedTime := sync.record.Status.ProvisionedTime
+
+		expectedProvisionedTime := metav1.Now()
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, expectedProvisionedTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
+
+		// mock changed MUR before syncing again
+		murtest.Modify(mur, murtest.StatusCondition(toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, "")))
```

Comment:
you don't handle an error here:
```suggestion
		err := murtest.Modify(mur, murtest.StatusCondition(toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, "")))
		require.NoError(t, err)
```

Reply:
apart from that - calling `murtest.Modify` on `mur` won't have any effect - the `prepareSynchronizer` uses a deep copy of the MUR.

Reply:
Thanks, I'm no longer using `murtest.Modify` because of your awesome suggestion below 🙂